### PR TITLE
Replaced MozFest banner with Thimble promo banner

### LINKED ISSUE #1166: AUGUST 31: Thimble promotion: homepage banner
@sabrinang - can you add assets?

### PR DESCRIPTION
**Pull request by mmmavis**
This fixes #1166.

Another PR for you to review @Pomax :dancers:

### CODE REVIEW

File: less/variables.less [hunk 1]
```diff
@@ -18,7 +18,7 @@
 @redSidebarColor: #BB3C33;
 @pinkSidebarColor: #D64262;
 @footerBackgroundColor: #f1f1f1;
-@mozFestPromoRed: #E12526;
+@thimblePromoGreen: #279761;
```
Comment (mmmavis) [inline less/variables.less]: Wondering if we should just give this a more general name instead (i.e., `promoBannerBackground`?) This way we don't have to update the variable name in both `less/variables.less` and `less/pages/home.less` every time we refresh the banner

Comment (Pomax) [inline less/variables.less]: yeah that seems a better idea. That we way we won't have to rename it in the future

File: pages/home.jsx [hunk 2]
```diff
@@ -314,15 +314,15 @@ var HomePage = React.createClass({
...
           </IconButtons>
         </HeroUnit>
-        <div className="row full-row mozfest-banner">
+        <div className="row full-row promo-banner">
           <div className="inner-container">
             <section>
               <Illustration
-                height={200} width={384}
-                src1x="/img/pages/home/Mozilla-Festival_Logo-Lockup_White.png" src2x="/img/pages/home/Mozilla-Festival_Logo-Lockup_White@2x.png"
+                height={190} width={190}
```
Comment (Pomax) [inline pages/home.jsx]: are we basing this off of the image itself, to make sure it's sized correctly? If so, we can *probably* get that off of the file by using a node module for checking metadata on png/jpg

Comment (mmmavis) [inline pages/home.jsx]: > are we basing this off of the image itself, to make sure it's sized correctly?

yup yup

> If so, we can probably get that off of the file by using a node module for checking metadata on png/jpg

Cool! Does that module automatically add `width` and `height` attributes to the underlying `` tag? (mind creating an issue for this?)

Comment (Pomax) [inline pages/home.jsx]: filed as https://github.com/mozilla/teach.webmaker.org/issues/1213

**Comment by Pomax** [comment]
this looks R+ to me, with a followup on possibly using a node module (or two) to find image dimensions so we don't have to hardcode in the future

File: pages/home.jsx (hunk 2 above)
Comment (mmmavis) [inline pages/home.jsx]: lovely!

**Pull request merged**